src/s2a_moms_tent.tw2

```diff
@@ -0,0 +1,287 @@
+
+Sun was going down. Ron would no doubt be stumbling in soon. No time like the present.
+
+I stepped forward and [[unlatched the door|Mom's Tent - den]].
+
+::Mom's Tent - den
+Things were different from when I last went home, however long ago that was. For one thing, the TV was off and the couch was empty.
+
+But it was clear Mom and Ron had been up to their usual bullshit recently: a fresh pile of cigarette butts littered the TV trays, and Ron’s prized bong still had [[the charred remains of a bowl packed into it|Mom's Tent - den 2]].
+
+::Mom's Tent - den 2
+I froze in place and listened intently.
+
+(click: "listened intently") [(transition: "dissolve") +(transition-time: 1s)[No noises. Just the tarp walls flapping in the evening breeze.]]
+
+(click: "No noises") [(transition: "dissolve")[I was alone, praise the fucking lord.]]
+
+(click: "praise the fucking lord") [(transition: "dissolve")[I walked over to the fridge a few feet away and scanned the shelves. Nothing much left — just half a ham sandwich on a paper plate, a twelve-pack of [[sodas|Mom's Tent - faygo]] and a lone [[can of beer|Mom's Tent - beer]].]]
+
+::Mom's Tent - faygo
+Ah, Redpop. Anna and I had this weirdo friend, Scott, who went absolutely bonkers for this shit. He loved to get turned wayyyy up on the sugar rush and rave about how delicious it was to drink “the blood of the innocent.”
+
+I kinda miss Scott. Wonder how he’s doing out there.
+
+I pulled a can of Redpop out of the box, cracked it open, and took [[a seat at the table|Mom's Tent - table]].
+
+::Mom's Tent - beer
+Ron’s pissy beer of choice: “The Great Milenkale”. It tastes worse than it smells, but given that I literally woke up in a toilet, I wasn’t feeling too choosy.
+
+I snatched a tallboy, cracked it open, and pulled out [[a seat at the kitchen table|Mom's Tent - table]].
+
+(set: $beerDrank to true)
+
+<!-- If using debug menu shortcuts, prompts player to enter their name -->
+::Mom's Tent - name prompt table
+{ (set: $playerName to (prompt: "Enter name:", ""))
+(if: (text: $playerName)'s length > 0)[Right, my name is $playerName. [[Moving on|Mom's Tent - table]].]
+}(else:)[No, [[that's not right|Mom's Tent - name prompt table]].]
+
+::Mom's Tent - table
+Anna’s disappearance was all I could think about, but it felt good to sit still for a moment. I realized suddenly that I ached all over. Come to think of it, what was I doing before I woke up in that nasty-ass bathroom? I strained to picture it, but it was all noise and disruption in my brain — [[nothing came to me|Mom's Tent - table hub]].
+
+(set: $cigarettesViewed to false)
+(set: $platesViewed to false)
+(set: $notepadViewed to false)
+
+::Mom's Tent - table hub
+I sighed, took another swig, and stared around the room.
+
+(if: $cigarettesViewed is false)[>>[[Cigarette butts galore. Looked like Mom was stressing hard again|Mom's Tent - cigarettes]].]
```

Comment:
The empty spaces that these options leave behind after you've viewed all three is a little distracting. We can probably find a good way to prevent this. Maybe we just move the `$tableDetailsSeen` if clause to line 76 and bump the rest down? We could also wrap that line in `{}` to prevent line breaks.